CMakeLists.txt

```diff
@@ -10,12 +19,22 @@ set(WEB3_DIR "${CMAKE_CURRENT_LIST_DIR}/webthree" CACHE PATH "The path to the we
 set(SOL_DIR "${CMAKE_CURRENT_LIST_DIR}/solidity" CACHE PATH "The path to the solidity libraries directory")
 
 # set cmake_policies
+
+testFileExists("${CMAKE_CURRENT_LIST_DIR}/webthree-helpers/cmake/EthPolicy.cmake")
+
 include(EthPolicy)
 eth_policy()
 
 # project name and version should be set after cmake_policy CMP0048
 project(cpp-ethereum VERSION "1.2.1")
 
+# test if files exist
+testFileExists("${CMAKE_CURRENT_LIST_DIR}/webthree-helpers/cmake/EthCompilerSettings.cmake")
```

Comment:
Ok, but when git clone without --recursive the cmake directory is downloaded but empty. I noticed that test only the directory it's not enough.